### ISSUE TITLE
NTR: Maternal zinc deficiency (General history, localization & modifiers)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7849]
New term request Maternal zinc deficiency (elthat_161002143611): In utero; pregnancy; embryopathy; maternal; mother; prenatal; prenatally

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000166
- label: Maternal zinc deficiency
- parents: [Pregnancy history (HP:0002686)](https://hpo.jax.org/app/browse/term/HP:0002686)
- pmid: PMID:26053136
- def: The mother of a fetus or index patient has a history of abnormally low zinc levels during the corresponding pregnancy.
- comment: Maternal zinc deficiency may increase the risk low birth weight (LBW) and small for gestational age (SGA) infants.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7849
- synonyms:
